src/test/utils/commandExecutor.test.ts

```diff
@@ -33,5 +33,5 @@ describe('CommandExecutor tests', () => {
         expect(exitCode).to.equal('0,');
         expect(stdout).to.contain('USAGE\n  $ sfdx provar');
         expect(stderr).to.contain('');
-    });
+    }).timeout(10000);
 });
```

Comment:
Again, this is a long delay in running the test suite.

Reply:
Done. I have reduced the timeout to 1s